apps/omg_utils/lib/omg_utils/http_rpc/response.ex

```diff
@@ -118,4 +129,26 @@ defmodule OMG.Utils.HttpRPC.Response do
 
     Map.merge(response, %{version: List.to_string(vsn) <> "+" <> @sha})
   end
+
+  # Not the most "beautiful way", but I'm just referencing
+  # how they're injecting the version
+  defp add_service_type(response) do
+    # TODO remove this hack when running releases
+    is_child_chain_running =
+      Enum.find(Application.started_applications(), fn
+        {:omg_child_chain_rpc, _, _} -> true
+        _ -> false
+      end)
+    
+    # Is type "watcher" or "childchain"
+    service_type =
+      if Code.ensure_loaded?(OMG.ChildChainRPC) and is_child_chain_running != nil do
+        "child_chain"
+      else
+        "watcher"
+      end
+    
+    # Inject it into the response code
+    Map.merge(response, %{type: service_type})
```

Comment:
I'm not sure `type` is appropriate word here. It could lead to confusion that it is `type of the response`...
I'm thinking of smth neutral e.g. `service_kind`, `responder`, `responder_service`... not sure

Reply:
`service_name`?

Reply:
I like `service_name`, lets go with that ⭐️